### DISCUSSION TITLE
Αλλαγή μέρας-ώρας για Συμπληρωματική ΟΣΣ3

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#124] [Announcements]
~~Θα γίνει συμπληρωματική ΟΣΣ3 Κυριακή 28 Ιανουαρίου 4μμ~~ #124 

Θα γίνει συμπληρωματική ΟΣΣ3 Κυριακή 28 Ιανουαρίου 7μμ 

https://hou.webex.com/meet/chorianopoulos.konstantinos

**Comment by iparthenios** [comment]
Δυστυχώς δεν θα μπορέσω να παρακολουθήσω. Λυπάμαι πραγματικά, αλλά υπήρχε προγραμματισμένη εκδήλωση... Κοπητηπιτα, της θεατρικής ομάδας που συμμετέχω.


ΚΑΛΗ ΔΥΝΑΜΗ!
